RestaurantManagementSystem/RestaurantManagementSystem/src/Core/Entities/RegistrationRequest.cs

```diff
@@ -0,0 +1,70 @@
+﻿using RestaurantManagementSystem.src.Infrastructure.Services;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+using shortid;
+using shortid.Configuration;
+
+namespace RestaurantManagementSystem.src.Core.Entities
+{
+    public class RegistrationRequest
+    {
+        
+        //props
+        public Guid Id;
+        [Required]
+        public string Name {get; }
+        [Required]
+        public string Email { get; }
+        [Required]
+        public string Address { get; }
+        [Required]
+        public ContactInfo ContactInfo { get;  }
+        public string VerificationCode { get; }
+        public RequestStatus.Status Status { get; private set; }
```

Comment:
Why it's nested? fix it to be like this
`public RequestStatus Status {get; private set;}`